install-dependencies.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+git clone -b hotfix/0.5 https://github.com/opencb/opencga.git
+cd opencga
+mvn install -DskipTests --settings ../.travis.settings.xml
```

Comment:
If the settings file is removed then `--settings ../.travis.settings.xml` must also be.